documentation/src/main/wiki/Server.md

```diff
@@ -214,6 +215,15 @@ The `repositories`, `splitter` and `assembler` verticles are specific to the def
 | `address`      | `String`         | &#10004;       | Event bus address of the **Knot** verticle |
 | `onTransition` | `KnotRouteEntry` |        | Describes routing to addresses of other Knots based on the transition trigger returned from current Knot.<br/>`"onTransition": { "go-d": {}, "go-e": {} }` |
 
+
+### AccessLogConfiguration options
+| Name  | Type  | Mandatory | Description  |
+|-------:|:-------:|:-------:  |-------|
+| `enabled`   | `boolean` |       | Enable/Disable access log. Default is `true` |
+| `immediate` | `boolean` |       | Log before request or after. Default is `false` - log after request |
```

Comment:
Maybe this property should be `mode` with values `"before"` and `"after"`?

Reply:
I'll keep it as it is now, as the same terminology is in the vertx access log handler used here.